ui/src/views/infra/changeStoragePoolScope.vue

```diff
@@ -0,0 +1,226 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+<template>
+  <a-spin :spinning="loading">
+    <div class="form-layout" v-ctrl-enter="handleSubmitForm">
+      <div class="form">
+        <a-form
+          :ref="formRef"
+          :model="form"
+          :rules="rules"
+          layout="vertical"
+          @submit="handleSubmitForm">
+          <a-alert type="warning">
+            <template #message>
+              <span v-html="resource.scope=='ZONE' ? $t('message.action.primary.storage.scope.cluster') : $t('message.action.primary.storage.scope.zone')"></span>
+            </template>
+          </a-alert>
+          <a-alert type="warning">
+            <template #message>
+              <span v-html="$t('message.warn.change.primary.storage.scope')"></span>
+            </template>
+          </a-alert>
```

Comment:
Do we need two different alerts? Aesthetically not looking very pleasing.

Reply:
message.action.primary.storage.scope.cluster is specific to cluster.
message.warn.change.primary.storage.scope is a generic msg about changing the storage pool scope (zone and cluster).

Is there a way we can combine these msgs into a single warning box?

Reply:
Not sure but can it be something like,
```
          <a-alert type="warning">
            <template #message>
              <span v-html="(resource.scope=='ZONE' ? $t('message.action.primary.storage.scope.cluster') : $t('message.action.primary.storage.scope.zone')) + '<br>' + $t('message.warn.change.primary.storage.scope')"></span>
            </template>
          </a-alert>
```

Need to check this actually works :)

Reply:
Its working :)